src/Core/src/Platform/Android/ImageSourcePartExtensions.cs

```diff
@@ -11,7 +12,7 @@ public static class ImageSourcePartExtensions
 		public static async Task<IImageSourceServiceResult?> UpdateSourceAsync(
 			this IImageSourcePart image,
 			View destinationContext,
-			IImageSourceServiceProvider services,
+			IMauiContext? mauiContext,
```

Comment:
Is there a reason for this API change? It is a public/braking API so we should avoid it. Also, using a maui context makes it a bit harder to unit test with custom services.